src/components/NavBarConsumerHome.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import '../styles/NavBar.css';
+import LogOutButton from './LogOutButton';
```

Comment:
your eslint rules should be forcing import organizatino....please add that too youor eslint configuration